packages/storage-documents/src/on-demand-page-scan-result.ts

```diff
@@ -36,6 +37,18 @@ export interface OnDemandPageScanResult extends StorageDocument {
     priority: number;
     itemType: ItemType.onDemandPageScanRunResult;
     batchRequestId: string;
+    notification?: ScanCompletedNotification;
+}
+
+export interface ScanCompletedNotification {
+    notificationUrl: string;
+    state?: NotificationState;
+    error?: NotificationError;
+}
+
+export interface NotificationError {
+    errorType: string;
```

Comment:
these should be in one of the known strings. (like ScanError). If not sure what all notification error types we will need, then just add one
NotificationErrorTypes= "HttpErrorCode"